.github/workflows/ci.yml

```
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  push:
+    branches:
+      # TODO: cover hotfix, release, master, etc?
+      - develop
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - "*"
+  schedule:
+    # Daily at 05:14
+    - cron: '14 5 * * *'
```

I just sort of shift this around when I copy it to try to not run all daily builds at the same time.  But, without actually keeping track of what's running when.  How about....  09:37 UTC.

```suggestion
    # Daily at 09:37
    - cron: '37 9 * * *'
```